README.md

```diff
@@ -1,6 +1,7 @@
 # gen3utils
 
-Utils for Gen3 commons management
+Utils for Gen3 commons management. Gen3utils includes a CLI and is intended to be used as an
+external facing tool for validating configuration.
```

Comment:
It does a little more than validating configs I believe. Like generating release notes etc.

Reply:
yes it does more - "validating configuration" in my message was an example

Reply:
I added more examples. Would we like more or are we fine with this?